lib/octo_events.ex

```diff
@@ -1,9 +1,6 @@
 defmodule OctoEvents do
-  @moduledoc """
-  OctoEvents keeps the contexts that define your domain
-  and business logic.
+  alias OctoEvents.{CreateEvent, ListEvent}
 
-  Contexts are also responsible for managing your data, regardless
-  if it comes from the database, an external API or others.
-  """
+  defdelegate create_event(params), to: CreateEvent
+  defdelegate list_event(issue_id), to: ListEvent
```

Comment:
```suggestion
  defdelegate list_events(issue_id), to: ListEvents
```

já que são vários eventos (: